Assets/Script/Gameplay/Visuals/Fret/FretArray.cs

```diff
@@ -92,12 +103,25 @@ public void Initialize(ThemePreset themePreset, GameMode gameMode,
             }
         }
 
-        public void InitializeColor(ColorProfile.IFretColorProvider fretColorProvider, bool leftyFlip)
+        public void InitializeColor(ColorProfile.IFretColorProvider fretColorProvider, bool leftyFlip, bool splitProTomsAndCymbals)
         {
             for (int i = 0; i < _frets.Count; i++)
             {
-                int index = i + 1;
-                if (DontFlipColorsLeftyFlip && leftyFlip)
+                /* This needs unique lefty flip logic because it's the one case where
+                 * the fret order is different from the color profile order
+                 */
+                int index = (splitProTomsAndCymbals) ? i switch {
+                    0 => leftyFlip ? 4 : 1,
+                    1 => leftyFlip ? 7 : 6,
+                    2 => leftyFlip ? 3 : 2,
+                    3 => leftyFlip ? 6 : 7,
+                    4 => leftyFlip ? 2 : 3,
+                    5 => leftyFlip ? 5 : 8,
+                    6 => leftyFlip ? 1 : 4,
+                    _ => throw new Exception("Unreachable.")
+                } : i + 1;
```

Comment:
Not the greatest fan of nesting switch expressions into ternaries like this, makes the control flow not as obvious at a glance.

```suggestion
                int index;
                if (splitProTomsAndCymbals)
                {
                    index = i switch {
                        0 => leftyFlip ? 4 : 1,
                        1 => leftyFlip ? 7 : 6,
                        2 => leftyFlip ? 3 : 2,
                        3 => leftyFlip ? 6 : 7,
                        4 => leftyFlip ? 2 : 3,
                        5 => leftyFlip ? 5 : 8,
                        6 => leftyFlip ? 1 : 4,
                        _ => throw new Exception("Unreachable.")
                    };
                }
                else
                {
                    index = i + 1;
                }
```